src/sentry/grouping/enhancer/enhancement-configs/newstyle@2023-01-11.txt

```diff
@@ -429,3 +429,18 @@ module:io.netty.* -app
 module:newrelic -app -group
 path:**/*newrelic*/** -app -group
 path:**/*LogRocket*/** -app -group
+
+## Unreal Engine
+
+### Unreal Engine internal assertion handling
+family:native stack.function:FDebug::CheckVerifyFailedImpl* ^-app -app v+app ^-group -group v+group
```

Comment:
[This](https://ivan-tustanivskyi.sentry.io/issues/6233924885/events/eb2737e5c27145405874d398f0485582/?project=4508533445754880) is an example of this rule.
<img width="866" alt="image" src="https://github.com/user-attachments/assets/44f9398d-ec0c-47cc-961c-42060baadddf" />

and [this](https://ivan-tustanivskyi.sentry.io/issues/6234280322/events/395f4d9f3bfb4bc5584d8f3af2467643/?project=4508533445754880):
<img width="757" alt="image" src="https://github.com/user-attachments/assets/3436dde8-3ec6-46e0-9a5f-fde09905fbc8" />

Reply:
@tustanivsky is this change also intentional?
<img width="701" alt="image" src="https://github.com/user-attachments/assets/e8ff712f-6cff-49c6-bd1d-bbe62c9b1709" />

<img width="494" alt="image" src="https://github.com/user-attachments/assets/f2a9fd85-2657-43dc-9b1d-5617de1c7079" />

Reply:
No, apparently `__start-thread` and `__pthread_start` were un-ignored because of the `v+app` in this rule. Is there a way we can still keep them ignored?

Reply:
We would have to add specific rules to keep them as non-in-app.